### PR TITLE
Add mariadb 10.11.2 and set it as the default mariadb package.

### PR DESCRIPTION
**Pull request by dlorenc**
10.11 is the latest LTS now, we had 10.6 previously. That's still LTS, but 10.11 will now last longer and is more recent.

To make this swap, I added a new package and renamed the old one. The new package provides mariadb with no version specifier.

Fixes: 

Related: 

#### For new package PRs only 

- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

#### For version bump PRs

- [x] The `epoch` field is reset to 0

### CODE REVIEW

File: mariadb-10.11.yaml [hunk 1]
```diff
@@ -0,0 +1,219 @@
+package:
+  name: mariadb-10.11
+  # Latest LTS
+  version: 10.11.2 # When bumping to 10.6.13, the patch for CVE-2022-47015 can be removed.
```
Comment (imjasonh) [inline mariadb-10.11.yaml:4]: This comment confuses me. When would we bump 10.11.2 (down) to 10.6.13?

edit: oh it's just copied.

File: mariadb-10.6.yaml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 package:
-  name: mariadb
-  # Latest LTS
+  name: mariadb-10.6
```
Comment (imjasonh) [inline mariadb-10.6.yaml:2]: Do we want this to also provide `mariadb=10.6.999`? (and throughout for the subpackages)

File: mariadb-10.11.yaml (hunk 1 above)
Comment (dlorenc) [reply mariadb-10.11.yaml:4]: D'oh! The comment worked too (I removed the patch), but I didn't remove the comment :)

File: mariadb-10.6.yaml (hunk 2 above)
Comment (dlorenc) [reply mariadb-10.6.yaml:2]: Fixed!

**Review by imjasonh — APPROVED** [review]

**Pull request merged**